frontend/src/components/filter/constants.js

```diff
@@ -156,6 +156,23 @@ export const FILTER_CONFIG = [
       />
     ),
   },
+  {
+    id: 'reportId',
+    display: 'Report ID',
+    conditions: SELECT_CONDITIONS,
+    defaultValues: EMPTY_TEXT_INPUT,
+    displayQuery: handleStringQuery,
+    renderInput: (id, condition, query, onUpdate, onApplyQuery) => (
```

Comment:
```renderInput: (id, condition, query, onApplyQuery) => (```

Refactored to drop the onUpdate since they were functionally the same

Reply:
Thanks 👍